draft_migration.sql

```diff
@@ -0,0 +1,61 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = events_api, public;
+
+CREATE TABLE IF NOT EXISTS webhooks (
```

Comment:
i wonder if this table should actually be events and then the event_type below becomes the event_name